docs/pages/web/tiledata.js

```diff
@@ -0,0 +1,179 @@
+
+export default function TileDataPage({ generatedDocGen }: {| generatedDocGen: DocGen |}): Node {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader
+        name={generatedDocGen?.displayName}
+        description={generatedDocGen?.description}
+        defaultCode={`
+function Example() {
+  return (
+    <TileData
+      title="Impressions"
+      value="10M"
+      selected
+      trend={{ value: 29, accessibilityLabel: 'Trending up' }}
+    />)
+}
+        `}
+      />
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+
+      <MainSection name="Usage guidelines">
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            title="When to use"
+            description={`
+        - When selecting and/or comparing categories with an accompanying chart or graph view that displays at-a-glance data for a user to quickly view key metrics
+      `}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            title="When not to use"
+            description={`
+        - When grouping Datapoints that aren't selectable
+        - For selectable information that is not part of a data visualization
+      `}
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Best practices">
+        {/* <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Use TileData paired with data visualization colors to make a visual connection between a chart and it’s related data"
+            defaultCode={`<TileData color="data-visualization-01" title="Blah" value="2" selected /> `}
+          />
+
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use colors outside of the Data Visualization palette. Please refer to the [Usage](/foundations/data_visualization/usage) and [Palette](/data_visualization/palette)."
+          />
+</MainSection.Subsection> */}
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="sm"
+            type="do"
+            description="Always present one tile in its selected state on default"
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" selected trend={{ value: 5, accessibilityLabel: 'Trending up' }} /> `}
+          />
+
+          <MainSection.Card
+            cardSize="sm"
+            type="don't"
+            description="Use it to present a single option, since they don't need to be selected. Use a [Datapoint](/web/datapoint) instead."
+            defaultCode={`<TileData color="01" title="Impressions" value="2M" trend={{ value: 1, accessibilityLabel: 'Trending up' }}/> `}
+          />
+        </MainSection.Subsection>
+        <MainSection.Subsection columns={2}>
```

Comment:
No visual examples for these?

Reply:
the team mentioned not doing visual examples on do's and don'ts if they're simple and don't need a visual for support